en/docs/developer-portal/manage-application.md

```diff
@@ -8,46 +8,150 @@ Let's get started!
+
+    ``` java tab="Format"
+    curl -k -X POST https://sts.choreo.dev/oauth2/token -d "client_id=<application_consumer_key>" -d "subject_token_type=urn:ietf:params:oauth:token-type:jwt" -d "grant_type=urn:ietf:params:oauth:grant-type:token-exchange" -d "subject_token=<idp_id_token>"
+    ```
+
+    ``` java tab="Example"
+    curl -k -X POST https://sts.choreo.dev/oauth2/token -d "client_id=ciwnWuwZfbcdzBUcnkhKvi_mcBUa" -d "subject_token_type=urn:ietf:params:oauth:token-type:jwt" -d "grant_type=urn:ietf:params:oauth:grant-type:token-exchange" -d "subject_token=eyJhbGciOiJFUzI1NiIsImtpZCI6IjE2In0.eyJhdWQiOiJodHRwczovL2FzLmV4YW1wbGUuY29tIiwiaXNzIjoiaHR0cHM6Ly9vcmlnaW5hbC1pc3N1ZXIuZXhhbXBsZS5uZXQiLCJleHAiOjE0NDE5MTA2MDAsIm5iZiI6MTQ0MTkwOTAwMCwic3ViIjoiYmNAZXhhbXBsZS5uZXQiLCJzY3AiOlsib3JkZXJzIiwicHJvZmlsZSIsImhpc3RvcnkiXX0.JDe7fZ267iIRXwbFmOugyCt5dmGoy6EeuzNQ3MqDek9cCUlyPhQC6cz9laKjK1bnjMQbLJqWix6ZdBI0isjsTA"
+    ```
+    
+    ```JSON tab="Response"
+        {
+        "access_token": "eyJ4NXQiOiJNV1E1TldVd1lXWmlNbU16WlRJek16ZG1NekJoTVdNNFlqUXlNalZoTldNNE5qaGtNR1JtTnpGbE1HSTNaRGxtWW1Rek5tRXlNemhoWWpCaU5tWmhZdyIsImtpZCI6Ik1XUTVOV1V3WVdaaU1tTXpaVEl6TXpkbU16QmhNV000WWpReU1qVmhOV000Tmpoa01HUm1OekZsTUdJM1pEbG1ZbVF6Tm1FeU16aGhZakJpTm1aaFl3X1JTMjU2IiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiJiNWViODFkMC01ZTMxLTQwZDgtYWY0MS03OWMwMjJlNTRhNTciLCJhdXQiOiJBUFBMSUNBVElPTl9VU0VSIiwiYXVkIjoibzRUWGxLQnowYWVJNEJ0d2NjY0V4bnVyNENVYSIsIm5iZiI6MTY2Mzg0MTM5MiwiYXpwIjoibzRUWGxLQnowYWVJNEJ0d2NjY0V4bnVyNENVYSIsInNjb3BlIjoiZGVmYXVsdCIsIm9yZ2FuaXphdGlvbiI6eyJ1dWlkIjoiZWMwZDE5OTgtZTRkNS00NjRlLWFjODgtOTg4ODM5NTYwZDVkIn0sImlzcyI6Imh0dHBzOi8vc3RzLmNob3Jlby5kZXY6NDQzL29hdXRoMi90b2tlbiIsImV4cCI6MTY2Mzg0NDk5MiwiaWRwX2NsYWltcyI6eyJhdXQiOiJBUFBMSUNBVElPTl9VU0VSIiwiYXV0aGVudGljYXRlZF9pZHAiOiJHb29nbGUiLCJuYW1lIjoid3NvMmUiLCJnaXZlbl9uYW1lIjoid3NvMiIsImZhbWlseV9uYW1lIjoid3NvMiIsImVtYWlsIjoid3NvMiJ9LCJpYXQiOjE2NjM4NDEzOTIsImp0aSI6ImM0MmRhNmM5LTYxYjEtNDUwNi1hMmYyLWY4OGE5MjQzNjk1MiJ9.M8FqUMDB1uvRTbz-AAxbl9af_72BLrSnTomGwasCzK4qRwVtV7MENLXFVzTZMW0ayBjPDQ8QeTmaky2LwAFKf0AYdGMtBYL0VLVWtBND7RCip_gxpa1k-Z5V4Cl5qcvKuLpySyE7E2QQqR9lPIHE-bm70nyWQheNMCEnPAKihyP772AEn3xRobHFGB_sidVuRAQm815GFE56NovKeBUM1YLmdnUgIrL1B9ho2q0g8aAI7959ORy9xQXLqxcYN-rd8uUnY5jWXPzxJHDDaozgHUK02IDKx0a-9Bf1Gy2Hj0DSQFRWZbPpfq7oMzCeRprqCYYkPkTjOz-LpyE9Ri-xTZNRbm7hua5PrRJjDD_EbsT8zYolRvIlwl7GcQOQuOtyxS4l_hQmb3DDmyFouv_P4Aknse4FvD8mSeXmvaTelyS5hBp0KNAoxkQBrRVgHVv_Jjyt2s9FwDhmLl4uS46x5Ca58X_wU6kEBP2hmfa98JHcfklog3gsxTm0T0JZlGs23zNqaf-ApC7NU9hWVZ6fwIX3CUzZNZmhF4caNrLJEqUnuxB5fHrHiwMNnoXf3WAILHj5gJPZo9OG18yymIdbRjHYrpD1ZQihIkPBTeMXwfRWIMDfBD3ezDKajktKqM8w1E-sFuGLHWyCqREN4XWA_jcZ6766BtTareAfKH171Ok",
+        "refresh_token": "a23h07f9-6ce5-3b7a-9d12-00e7f82f4f46",
+        "issued_token_type": "urn:ietf:params:oauth:token-type:jwt",
+        "scope": "default",
+        "token_type": "Bearer",
+        "expires_in": 3600
+        }
+    ```
```

Comment:
Are the "Example" and "Response" the result of your test or did you confirm these results?

Reply:
Yes I did run and test.